app/components/circularDisplay/AstroDataContext.tsx

```diff
@@ -29,6 +29,7 @@ export const AstroDataContext = createContext<AstroDataContextProps>({})
 /**
  * An Astro Flavored Markdown enriched link.
  */
+// eslint-disable-next-line react/display-name
```

Comment:
Why was this addition necessary?

Reply:
when I moved the component into the components directory, it is judged as a component definition and causes this warning:
<img width="1040" alt="Screenshot 2024-10-22 at 1 58 18 PM" src="https://github.com/user-attachments/assets/4d472f07-67bc-4284-9fcb-b75dba14a55f">

Reply:
I was trying to make the move have no changes to the code if possible